### DISCUSSION TITLE
Can't acessess any kind of account

### DISCUSSION THREAD

**Discussion by MigsRS123** [vercel/community#3902] [Account Issue] [answered]
### Question

Hi, I'm trying to log in with a hobby account with GitHub, but it says that the account is blocked, I tried with other emails but either they said that it was blocked or that it couldn't complete the process.

**Comment by leerob** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
